core/store/orm/orm_test.go

```diff
@@ -784,7 +784,7 @@ func TestORM_GetLastNonce_Valid(t *testing.T) {
 	t.Parallel()
 	config, cleanup := cltest.NewConfig(t)
 	defer cleanup()
-	config.Set("ENABLE_BULLETPROOF_TX_MANAGER", false)
+	config.Set("ENABLE_BULLETPROOF_TX_MANAGER", false) // TODO - remove with test
```

Comment:
Can this go?

Reply:
these tests will get removed when we get rid of the legacy TXM (coming next!)